corehq/apps/app_manager/suite_xml.py

```diff
@@ -1400,6 +1400,7 @@ def _schedule_detail_variables(module, detail, detail_column_infos):
         has_schedule_columns = any(ci.column.field_type == FIELD_TYPE_SCHEDULE for ci in detail_column_infos)
         has_schedule = getattr(module, 'has_schedule', False)
         if (has_schedule and module.all_forms_require_a_case() and has_schedule_columns):
+            yield DetailVariable(name=SCHEDULE_DATE_CASE_OPENED, function="date_opened")  # date case is opened
```

Comment:
Hah, you are right. I actually had a different, less descriptive variable name, which I then changed. :smile_cat: